src/index.js

```diff
@@ -1,29 +1,7 @@
-import useManifest from './hooks/useManifest'
 import Manifest from './components/Manifest'
-import Debug from './components/Debug'
-import useCell from './hooks/useCell'
-import useHeaderCell from './hooks/useHeaderCell'
-import usePager from './hooks/usePager'
-import DefaultTable from './components/DefaultTable'
-import DefaultControls from './components/DefaultControls'
-import DefaultControlsPageSizer from './components/DefaultControls/PageSizer'
-import DefaultControlsStatus from './components/DefaultControls/Status'
-import ButtonPager from './components/ButtonPager'
-import { ASCENDING, DESCENDING, NOT_SORTED } from './constants/sortDirections'
+import Filter from './components/Filter'
 
 export {
-  useManifest,
-  useCell,
-  useHeaderCell,
-  usePager,
   Manifest,
-  Debug,
```

Comment:
Might want to continue to expose the debug as a passthrough to use-manifest's Debug

Reply:
I added a debug prop to handle this.  I am not sure it would be easy to insert Debug otherwise